client/src/components/Bookings/NextBooking.tsx

```diff
@@ -0,0 +1,29 @@
+import BookingCard from './BookingCard';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@mui/material/Paper';
+import { useRequest } from '../../context/useBookingContext';
+import useStyles from './useStyles';
+import { checkIfUndefined } from '../../helpers/utils/checkIfUndefined';
+import moment from 'moment';
+
+const NextBooking = (): JSX.Element => {
+  const classes = useStyles();
+  const { bookings } = useRequest();
+
+  // find next Booking
+  const nextBooking = checkIfUndefined(bookings.find((booking) => booking.start.getTime() > Date.now()));
+
+  // format start date
+  const startDate = moment(nextBooking.start).format('MMMM Do YYYY');
```

Comment:
Comments explaining trivial operations don't usually add much to code clarity but instead hinder readability. We can get rid of them here, the variable naming clearly conveys what these lines are doing 👍🏼